src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -105,7 +105,7 @@ class IssueAlertOptions extends AsyncComponent<Props, State> {
       ...super.getDefaultState(),
       conditions: [],
       intervalChoices: [],
-      alertSetting: `${Actions.CREATE_ALERT_LATER}`,
+      alertSetting: Actions.CREATE_ALERT_LATER.toString(),
```

Comment:
Why the string cast change?

Reply:
Mmm, I feel like this is a lot more obvious than understanding what's going on with

```
`${maybeNotString}`
```

I only included this in this PR so I didn't have to make another one, I can be less lazy but 🤷‍♂️